packages/smooth_app/lib/pages/preferences/user_preferences_account.dart

```diff
@@ -216,6 +217,33 @@ class _UserPreferencesPageState extends State<UserPreferencesSection> {
     );
   }
 
+  Future<int?> _getMyCount(
```

Comment:
Please put private or not important methods rather at the end of the code, after the `build` method anyway.